lib/helpers/results_helpers.js

```diff
@@ -321,7 +321,7 @@ module.exports = class ResultsHelpers {
       }
       return quantityResult;
     } else if (result && result._type && result._type.includes('QDM::')) {
-      const description = result.description ? `${result.description}\n` : '';
+      const description = result.description ? `${result.description}\n` : `${result._type.replace('QDM::', '')}\n`;
```

Comment:
add comment. not clear to me why the `_type` is a good replacer for description when description not present.